src/test/java/seedu/address/model/transaction/AmountTest.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.model.transaction;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class AmountTest {
+
+    @Test
+    public void isValidAmount() {
```

Comment:
Think the naming for this test can be improved

Reply:
👌